.github/workflows/ci.yml

```
@@ -257,14 +257,17 @@ jobs:
     # pnpmで依存関係をインストール
     - name: Install Dependencies
       run: pnpm install --frozen-lockfile
-
+    
```

yamllint

[error] trailing spaces

(trailing-spaces)